Tests/Shared.Specs/ExceptionAssertionSpecs.cs

```diff
@@ -889,12 +888,132 @@ public void When_no_exception_should_be_thrown_and_none_was_it_should_not_throw(
             // Arrange
             //-----------------------------------------------------------------------------------------------------------
             var foo = A.Fake<IFoo>();
-
             //-----------------------------------------------------------------------------------------------------------
             // Act / Assert
             //-----------------------------------------------------------------------------------------------------------
             foo.Invoking(f => f.Do()).Should().NotThrow();
         }
+
+        #pragma warning disable CS1998
+        [Fact]
+        public void NotThrowAfter_when_subject_is_async_it_should_not_throw()
```

Comment:
@dennisdoomen usually has some opinions on naming tests

Reply:
I tried to stick to the naming pattern found in the tests. If a test covers a single method, I think its nice to reflect this in the name. This is also done in this test suite, e.g. in the tests for `WithInnerExceptionExactly`